src/net/tls.cc

```diff
@@ -987,7 +1064,7 @@ void tls::reloadable_credentials<tls::server_credentials>::rebuild(const credent
 }
 
 template <>
-const tls::certificate_credentials &tls::reloadable_credentials<tls::server_credentials>::as_certificate_credentials() const noexcept{
+const tls::certificate_credentials& tls::reloadable_credentials<tls::server_credentials>::as_certificate_credentials() const noexcept{
```

Comment:
Move the fix to the patch that introduced the problem please.

Reply:
Hm, ya my mistake